### PR TITLE
Add Traffic Rule switches to UniFi Network

### PR DESCRIPTION
**Pull request by ViViDboarder**
## Proposed change

Adds documentation for UniFi Traffic Rule switches

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118821
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a "Traffic Rules" section in the Unifi integration documentation, detailing the management of traffic rules with toggle functionality.
- **Documentation**
	- Enhanced clarity and usability of the Unifi integration documentation by integrating new content on traffic rules.

### CODE REVIEW

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Comment by ViViDboarder** [comment]
Oh, I forgot about this PR. Someone took the parent PR listed here and continued it for me when I got busy. That PR was merged but I didn’t see a documentation PR merged with it. This PR may still be valid. 

Here’s the merged core PR: https://github.com/home-assistant/core/pull/118821

**Comment by Kane610** [comment]
I changed the referred issue to the new one

**Review by Kane610 — DISMISSED** [review]

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -126,6 +126,10 @@ Entities appear for each port forwarding rule.
 
 Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfiguration of affected access points, limiting access to all WLANs exposed by the access point.
 
+### Traffic Rules
+
+Entities appear for each Traffic Rule. This allows toggling of rules on and off.
```
Comment (c0ffeeca7) [inline source/_integrations/unifi.markdown:131]:
```suggestion
Entities appear for each Traffic Rule. This allows toggling the rules on and off.
```

Comment (c0ffeeca7) [inline source/_integrations/unifi.markdown]:
```suggestion
Entities appear for each Traffic Rule. This allows toggling rules on and off.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ViViDboarder and @Kane610 :+1:

**Pull request merged**